ui/app/pages/swaps/select-quote-popover/index.scss

```diff
@@ -143,7 +132,37 @@
   &__receiving {
     display: flex;
     flex-direction: column;
-    width: 102px;
+    width: 88px;
+  }
+
+  &__network-fees {
+    width: 99px;
+  }
+
+  &__quote-source {
+    width: 28px;
+    display: flex;
+    align-items: flex-end;
+    margin-right: 3px;
```

Comment:
The `width` property of each column was adjusted, but this `margin-right` was the biggest change. It went from `18px/32px` to `3px`, but in reality the change is less than that - it was never at the full specified width in practice because the container wasn't wide enough to contain all of these children at their specified width. Also the 8px margin makes this closer to ~11px now in practice.